pusher/signature.py

```diff
@@ -1,27 +1,36 @@
 # -*- coding: utf-8 -*-
 
+from __future__ import (
+    print_function,
+    unicode_literals,
+    absolute_import,
+    division)
+
 import hashlib
 import hmac
 import six
 
+
 try:
     compare_digest = hmac.compare_digest
+
 except AttributeError:
     # Not secure when the length is supposed to be kept secret
     def compare_digest(a, b):
         if len(a) != len(b):
             return False
-        return reduce(lambda x, y: x | y, [ord(x) ^ ord(y) for x, y in zip(a, b)]) == 0
+
+        return reduce(
+            lambda x, y: x | y, [ord(x) ^ ord(y) for x, y in zip(a, b)]) == 0
```

Comment:
this is not very legible (took me a couple of minutes to understand this one line) but, still, a smart piece of code!